packages/file_selector/file_selector_windows/example/windows/CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
Once this is checked in we should probably land a really quick update that updates this to the latest version of the template. Apologies for the delayed review.